gradle/libs.versions.toml

```diff
@@ -2,7 +2,7 @@
 kotlinVersion = "1.8.21"
 
 # Logging
-slf4jVersion = { strictly = "1.7.36" }
+slf4jVersion = "2.0.12"
```

Comment:
We probably need to bump the version of `corda-api` on this instance, as this is represents a substantial change, which is arguably of the breaking nature.